lms/djangoapps/course_home_api/dates/v1/serializers.py

```diff
@@ -18,6 +19,7 @@ class DateSummarySerializer(serializers.Serializer):
     description = serializers.CharField()
     learner_has_access = serializers.SerializerMethodField()
     link = serializers.SerializerMethodField()
+    title_html = serializers.CharField()
```

Comment:
I left a comment about this in the frontend, but we don't want to be sending title_html since it represents arbitrary HTML which can be a security concern. Instead, we want to use other bits of information to determine whether the title should be a link or not. This can be seen in the Dates Tab code that already exists for the MFE.